docs/download/repositories.md

```diff
@@ -193,16 +193,18 @@ Microsoft Azure cloud service containing ICGC data.
 | Property         | Value                                                                                                                                   |
 | :--------------- | --------------------------------------------------------------------------------------------------------------------------------------- |
 | Name             | Microsoft Azure                                                                                                                 |
-| Contact          | <mailto:dcc-support@icgc.org>                                                                                                      |
+| Contact          | [ICGC ARGO Contact page](https://platform.icgc-argo.org/contact)                                                                        |
 | Repository type  | Cloud                                                                                                      |
 | ICGC Portal Page | [Portal](https://dcc.icgc.org/repositories?filters=%7B%22file%22:%7B%22repoName%22:%7B%22is%22:%5B%22Azure%20-%20Toronto%22%5D%7D%7D%7D&files=%7B%22from%22:1%7D) |
 | Download Client  | [Tarball](/software/download/#score-client), [Docker](https://hub.docker.com/r/overture/score/) |
 | Repo Code        | `azure`                                                                                                                          |
 
 ### Obtaining Data Access
-Follow the procedure outlined at the [DACO page](https://icgc.org/daco).
+Follow the procedure outlined at the [DACO page][1].
```

Comment:
let's change to 

To obtain access to Controlled Data, learn more about the [DACO application process, ](https://docs.icgc-argo.org/docs/data-access/daco/applying)or [start a DACO application here](https://daco.icgc-argo.org/).